testing/alignment.cu

```diff
@@ -210,14 +210,31 @@ void test_aligned_type()
 DECLARE_UNITTEST(test_aligned_type);
 
 template <std::size_t Len, std::size_t Align>
-void test_aligned_storage_instantiation()
+void test_aligned_storage_instantiation(thrust::detail::true_type /* Align is valid */)
 {
     typedef typename thrust::detail::aligned_storage<Len, Align>::type type;
     ASSERT_GEQUAL(sizeof(type), Len);
     ASSERT_EQUAL(THRUST_ALIGNOF(type), Align);
     ASSERT_EQUAL(thrust::detail::alignment_of<type>::value, Align);
 }
 
+template <std::size_t Len, std::size_t Align>
+void test_aligned_storage_instantiation(thrust::detail::false_type /* Align is invalid */)
+{
+  std::cerr << "Skipping test_aligned_storage_instantiation<Len=" << Len
+            << ", Align=" << Align << "> (MaxAlign="
+            << THRUST_ALIGNOF(thrust::detail::max_align_t) << ").\n";
```

Comment:
Let's probably just make this a comment.

Reply:
Let's open an issue and do this as a separate commit to avoid blocking this change from landing.